src/main/java/uk/gov/companieshouse/chipsrestinterfacesconsumer/service/impl/MessageProcessorServiceImpl.java

```diff
@@ -40,39 +41,53 @@
     private MessageProducer messageProducer;
 
     @Override
-    public void processMessage(String consumerId, ChipsRestInterfacesSend message) {
+    public void processMessage(String consumerId,
+                               ChipsRestInterfacesSend message,
+                               List<String> failedMessageIds) {
+
         Map<String, Object> logMap = new HashMap<>();
         logMap.put("Message", message.getData());
         logMap.put("Message Consumer ID", consumerId);
         try {
             chipsRestClient.sendToChips(message, consumerId);
         } catch (HttpStatusCodeException hsce) {
             logMap.put("HTTP Status Code", hsce.getStatusCode().toString());
-            handleFailedMessage(message, hsce, logMap);
+            handleFailedMessage(message, hsce, logMap, failedMessageIds);
         } catch (Exception e) {
-            handleFailedMessage(message, e, logMap);
+            handleFailedMessage(message, e, logMap, failedMessageIds);
         }
     }
 
-    private void handleFailedMessage(ChipsRestInterfacesSend message, Exception e, Map<String, Object> logMap) {
+    private void handleFailedMessage(ChipsRestInterfacesSend message,
+                                     Exception e,
+                                     Map<String, Object> logMap,
+                                     List<String> failedMessageIds) {
+
         var messageId = message.getMessageId();
         logger.errorContext(messageId, SEND_FAILURE_MESSAGE, e, logMap);
 
         if (runAppInErrorMode) {
             message.setAttempt(1);
             messageProducer.writeToTopic(message, retryTopicName);
+            if(failedMessageIds != null) {
```

Comment:
this is one for @ICoxCH and @sroberts2 as well. Should we send the message to FESS if the error consumer fails to call chips? it will go onto the retry topic and could potentially succeed there. The process for the main consumer is to send to FESS if all retries fail.

Reply:
I would say we should only be notifying FESS of messages being added to the error topic. 
In which case we shouldn't notify support here.

If it fails all retries again, it will notify FESS when added back to the error topic, doing a notification here will likely just create duplicate notifications

Reply:
Removed duplicate slack call.